terrautils/lemnatec.py

```diff
@@ -50,6 +50,8 @@
 SENSOR_VNIR = "VNIR"
 SENSOR_WEATHER = "weather"
 
+LEMNATEC_LOCAL_CACHE_FOLDER = '/home/extractor/lemnatec/'
```

Comment:
we may want to allow this to be overridden by environment variable

Reply:
this should now be done, along with a method to set that environmental variable

Reply:
I think you can simplify your code here, you can remove the _ENV copy of this and the if statements to check for it and instead use something like:
https://github.com/terraref/terrautils/blob/master/terrautils/betydb.py#L36
```
LEMNATEC_LOCAL_CACHE_FOLDER = os.environ.get('LEMNATEC_LOCAL_CACHE', '/home/extractor/lemnatec/')
```
This will use env var value of LEMNATEC_LOCAL_CACHE (e.g. set in Docker) if it exists, but revert to /home/extractor/lematec if it doesn't. then you can just keep referring to your original var.

Reply:
Done